### DISCUSSION TITLE
I would like to help with project's website

### DISCUSSION THREAD

**Discussion by vitormontana** [SpiralLinux/SpiralLinux-project#311] [General]
I am a frontend web dev, and I would like to help improve the website and make it look modern

**Comment by geckolinux** [comment]
Hi there, thanks a lot for offering. What changes do you propose? I'm fairly picky about small details, so I can't guarantee I'll accept the changes, but you can fork the website code here:
https://github.com/SpiralLinux/spirallinux.github.io/fork
